p9/messages.go

```diff
@@ -2027,6 +2027,9 @@ type registry struct {
 // This takes, and ignores, a message tag so that it may be used directly as a
 // lookuptagAndType function for recv (by design).
 func (r *registry) get(_ tag, t msgType) (message, error) {
+	if t >= math.MaxUint8 {
```

Comment:
t is a uint8. wouldn't it be easier to do `factories [math.MaxUint8+1]msgFactory`? (I think either is valid)

Reply:
the exact same comment on the gvisor side :smile:  https://github.com/google/gvisor/pull/3317#discussion_r458362078

I'll update both :)

Reply:
i learned a lot from them :)